content/apprenticeships/techtonic.md

```diff
@@ -2,7 +2,7 @@
 company: "Techtonic"
 description: "Lots of schools teach code. At Techtonic, we empower our apprentices with hands-on development of real client projects to learn how to apply their knowledge to solve business problems."
 image: "techtonic.jpg"
-link: "https://www.techtonic.com/"
+link: "https://www.techtonic.com/apprenticeships/overview/"
```

Comment:
Hey @aaronclarkcodes this location leads to a 404 page. Seems the ending slash is the problem. If you remove it, the link works properly.

Reply:
@benjaminmodayil changed